medusa/__main__.py

```diff
@@ -994,6 +995,18 @@ def initialize(self, console_logging=True):
                 logger.debug(u"Unable to find '{config}', all settings will be default!", config=app.CONFIG_FILE)
                 self.save_config()
 
+            if app.SUBTITLES_ERASE_CACHE:
+                try:
+                    for cache_file in ['application.dbm', 'subliminal.dbm']:
+                        file_path = os.path.join(app.CACHE_DIR, cache_file)
+                        if os.path.isfile(file_path):
+                            logger.debug(u"Removing subtitles cache file: {cache_file}", cache_file=file_path)
+                            os.remove(file_path)
+                except OSError as e:
+                    logger.debug(u"Unable to remove subtitles cache files. Error: {error}", error=e)
```

Comment:
Yeah that was also my first impression. But with a warning, a user should know from the warning which action it should take.